randovania/games/dread/json_data/Dairon.txt

```diff
@@ -566,7 +589,7 @@ Extra - asset_id: collision_camera_007
   * Extra - start_point_actor_name: powergeneratorengine_001
```

Comment:
For visualization purposes, could you adjust the coordinates of the start point so it doesn't overlap with the event? At the moment, it looks like they're the same node, and I thought the event had no connections coming out because of it. Maybe set X=700 or something like that.